Vagrantfile

```diff
@@ -26,6 +26,9 @@ Vagrant.configure('2') do |config|
     touch /home/vagrant/.bashrc
     grep -Fxq 'source /home/vagrant/openbalena/.openbalenarc' /home/vagrant/.bashrc || echo 'source /home/vagrant/openbalena/.openbalenarc' >> /home/vagrant/.bashrc
 
+    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | bash
+    source "/home/vagrant/.nvm/nvm.sh" # This loads nvm
+    nvm install node && nvm use node
```

Comment:
Cab you please pin it to a specific version because otherwise it’s a moving target? The latest LTS is 10.15.0.

Reply:
Agreed - pinned to 10.15.0